packages/client/hmi-client/src/api/api.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import useAuthStore from '../stores/auth';
+
+const API = axios.create({
+	baseURL: '/api'
+});
+
+// Hook in bearer tokens
+// See
+// - https://medium.com/swlh/handling-access-and-refresh-tokens-using-axios-interceptors-3970b601a5da
+API.interceptors.request.use((config) => {
+	const auth = useAuthStore();
+
+	if (config.headers) {
+		config.headers.Authorization = `Bearer ${auth.token}`;
+	} else {
+		config.headers = { Authorization: `Bearer ${auth.token}` };
+	}
+	return config;
```

Comment:
Do we need an error handler here? or passing that along to whoever uses it?

Reply:
I added a stubbed error handler so people know where it is, we can extend it if necessary